dev-docs/bidders/yieldone.md

```diff
@@ -1,18 +1,18 @@
 ---
 layout: bidder
-title: YIELDONE
-description: Prebid YIELDONE Bidder Adaptor
+title: YieldOne
+description: Prebid YieldOne Bidder Adaptor
 biddercode: yieldone
 media_types: banner, video
-userIds: identityLink, imuid
+userIds: identityLink, imuid, AudienceOne ID
 pbjs: true
 pbs: true
```

Comment:
Thanks for updating the docs. Could you add the flag multiformat_supported (introduced in #3694 ) as well?

It allows sales houses / publishers to better understand the behaviour of the bid adapter for multi format ad units.

Reply:
@muuki88 
Thank you for your review. I added multiformat_supported field.
Could you confirm, please?

Reply:
@muuki88 
Hi, How is the progress here? Could you review, please?